src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java

```diff
@@ -113,4 +118,32 @@ Boolean isKnownParticipant(
             @Path( ENTITY_KEY_ID ) UUID participantEntityKeyId,
             @Query( FILE_TYPE ) FileType fileType
     );
+
+    /**
+     * Update chronicle_used_by associations when apps usage survey is submitted
+     *
+     * @param studyId               - the study id
+     * @param participantId         - participantId
+     * @param neighborEntityDetails - neighbor entities of participant
+     * @return number of updated associations
+     */
+    @POST ( BASE + PARTICIPANT_PATH + DATA_PATH + STUDY_ID_PATH + PARTICIPANT_ID_PATH + APPS )
+    public Integer updateAppsUsageAssociationData(
+            @PathVariable  (STUDY_ID) UUID studyId,
+            @PathVariable (PARTICIPANT_ID) String participantId,
+            @RequestBody Set<NeighborEntityDetails> neighborEntityDetails
+    );
+
+    /**
+      * Get all apps usage data associated with a participant filtered by current date
+      * @param studyId              - the studyId
+      * @param participantId        - the participant
+      * @return a list of neighbor entities
+      */
+    @GET ( BASE + PARTICIPANT_PATH + DATA_PATH + STUDY_ID_PATH + PARTICIPANT_ID_PATH + APPS)
+    List<NeighborEntityDetails> getParticipantAppsUsageData(
+            @PathVariable(STUDY_ID) UUID studyId,
+            @PathVariable (PARTICIPANT_ID) String participantId
```

Comment:
I think these need to be `@Path`, not `@PathVariable`. Can you double check this?

Reply:
@path is from retrofit2, @pathVariable is from springframework. I believe they are interchangeable.